app/components/ScoreGauge.tsx

```
@@ -0,0 +1,62 @@
+    const percentage = score / 100;
+
+    useEffect(() => {
+        if (pathRef.current) {
+            setPathLength(pathRef.current.getTotalLength());
+        }
+    }, []);
+
+    return (
+        <div className="flex flex-col items-center">
+            <div className="relative w-40 h-20">
+                <svg viewBox="0 0 100 50" className="w-full h-full">
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)